apps/omg/lib/omg/state.ex

```diff
@@ -117,6 +118,14 @@ defmodule OMG.State do
     {:noreply, state}
   end
 
+  def handle_info(:send_metrics, state) do
```

Comment:
Do we have a way not to block `State` to collect this❓ 
Is it a good practice to get a current state and do computation asynchronously 
(CC: @paulperegud , @InoMurko )

Reply:
Depends how busy the process is.